src/main/resources/assets/create/lang/es_es.json

```diff
@@ -0,0 +1,1797 @@
+	"block.create.green_sail": "Vela verde",
+	"block.create.green_seat": "Asiento verde",
+	"block.create.green_valve_handle": "Asa de válvula verde",
+	"block.create.hand_crank": "Manivela",
+	"block.create.honey": "Miel",
+	"block.create.horizontal_framed_glass": "Cristal con marco horizontal",
+	"block.create.horizontal_framed_glass_pane": "Panel de cristal con marco horizontal",
+	"block.create.hose_pulley": "Polea de manguera",
+	"block.create.item_drain": "Drenaje de elementos",
+	"block.create.jungle_window": "Ventana de jungla",
+	"block.create.jungle_window_pane": "Panel de ventana de jungla",
+	"block.create.large_cogwheel": "Rueda dentada grande",
+	"block.create.layered_andesite": "Andesita estratificada",
+	"block.create.layered_dark_scoria": "Escoria oscura estratificada",
+	"block.create.layered_diorite": "Diorita estratificada",
+	"block.create.layered_dolomite": "Dolomita estratificada",
+	"block.create.layered_gabbro": "Gabro estratificado",
+	"block.create.layered_granite": "Granito estratificado",
+	"block.create.layered_limestone": "Piedra caliza estratificada",
+	"block.create.layered_scoria": "Escoria estratificada",
+	"block.create.layered_weathered_limestone": "Caliza erosionada estratificada",
+	"block.create.light_blue_sail": "Vela azul claro",
+	"block.create.light_blue_seat": "Asiento azul claro",
+	"block.create.light_blue_valve_handle": "Asa de válvula azul claro",
+	"block.create.light_gray_sail": "Vela gris claro",
+	"block.create.light_gray_seat": "Asiento gris claro",
+	"block.create.light_gray_valve_handle": "Asa de válvula gris claro",
+	"block.create.lime_sail": "Vela lima",
+	"block.create.lime_seat": "Asiento lima",
+	"block.create.lime_valve_handle": "Asa de válvula lima",
+	"block.create.limesand": "Arena caliza",
+	"block.create.limestone": "Piedra caliza",
+	"block.create.limestone_bricks": "Ladrillos de piedra caliza",
+	"block.create.limestone_bricks_slab": "Losa de ladrillos de piedra caliza",
+	"block.create.limestone_bricks_stairs": "Escaleras de ladrillos de piedra caliza",
+	"block.create.limestone_bricks_wall": "Pared de ladrillos de piedra caliza",
+	"block.create.limestone_cobblestone": "Adoquínes de piedra caliza",
+	"block.create.limestone_cobblestone_slab": "Losa de adoquines de piedra caliza",
+	"block.create.limestone_cobblestone_stairs": "Escaleras de adoquines de piedra caliza",
+	"block.create.limestone_cobblestone_wall": "Pared de adoquines de piedra caliza",
+	"block.create.limestone_pillar": "Pilar de piedra caliza",
+	"block.create.linear_chassis": "Chasis lineal",
+	"block.create.lit_blaze_burner": "Lit Blaze Burner",
+	"block.create.magenta_sail": "Vela magenta",
+	"block.create.magenta_seat": "Asiento magenta",
+	"block.create.magenta_valve_handle": "Asa de válvula magenta",
+	"block.create.mechanical_arm": "Brazo mecánico",
+	"block.create.mechanical_bearing": "Rodamiento mecánico",
+	"block.create.mechanical_crafter": "Mecanizador",
```

Comment:
@muriplz I am in doubt with this word, how can we call it? Crafteador Mecánico? idk

Reply:
@muriplz Thank you for your help!

Reply:
hard to tell, i thought of these two: 
    -Ensambladora.
    -Mesa de trabajo automática/ mecánica.
Crafteador automático also fits :D

Reply:
@muriplz Auto-ensamblador mecánico? 🤔